Sources/HTTP/Message/HTTPHeaderStorage.swift

```diff
@@ -14,29 +14,25 @@ final class HTTPHeaderStorage {
 
     /// Creates a new `HTTPHeaders` with default content.
     static func `default`() -> HTTPHeaderStorage {
-        let storageSize = 64
-        let buffer = MutableByteBuffer(start: .allocate(capacity: storageSize), count: storageSize)
-        memcpy(buffer.start, defaultHeaders.start, defaultHeadersSize)
-        let view = ByteBuffer(start: buffer.baseAddress, count: defaultHeadersSize)
+        let buffer = MutableByteBuffer.allocate(capacity: defaultHeadersSize)
+        _ = buffer.initialize(from: defaultHeaders)
```

Comment:
`_ = ` could be avoided with an `assert` done either after this line or, better, in a generic extension. `assertInitialize`